app.js

```diff
@@ -1,9 +1,11 @@
 const express = require("express");
 const app = express();
-const { getCategories } = require("./controllers/controller");
+const { getCategories, getReviews } = require("./controllers/controller");
 
 app.get("/api/categories", getCategories);
 
+app.get("/api/reviews", getReviews);
+
 app.all("*", (req, res, next) => {
```

Comment:
you don't need next with this piece of middleware

Reply:
Thank you will ammend